libraries/AP_RangeFinder/AP_RangeFinder_MAVLink.cpp

```diff
@@ -77,4 +78,13 @@ void AP_RangeFinder_MAVLink::update(void)
     }
 }
 
+bool AP_RangeFinder_MAVLink::get_signal_quality_pct(uint8_t &quality_pct) const
+{
+    if (status() != RangeFinder::Status::Good) {
+        return false;
+    }
+    quality_pct = signal_quality;
```

Comment:
```suggestion
    if (signal_quality <= 1) {
        return false;
    }
    quality_pct = signal_quality * 99 / 100;
```

not sure on that scaling, needs to scale from (1-100) to (0-100)

Reply:
this is not done in handle_msg(), above